samples/highcharts/dragdrop/drag-xrange/demo.js

```diff
@@ -6,13 +6,18 @@ var setDragStatus = function (status) {
 Highcharts.chart('container', {
     chart: {
         animation: false,
-        type: 'xrange'
+        type: 'xrange',
+        zoomType: 'x'
     },
 
     title: {
         text: 'Highcharts draggable xrange demo'
     },
 
+    subtitle: {
+        text: 'Test zoom and dragging/resizing of points'
+    },
+
```

Comment:
Internal testing notes shouldn't bloat the demo (this is available from the API). Instead of this, it would be better if we add the instructions to a `test-notes.md` file.